virttest/utils_test/libvirt.py

```diff
@@ -2371,19 +2372,47 @@ def create_tpm_dev(params):
     return tpm_dev
 
 
-def get_vm_device(vmxml, dev_tag, index=0):
+def get_vm_device(vmxml, dev_tag, dev_dict={}, index=0):
```

Comment:
Could you please update the default value of dev_dict as None? It may lead to some unexpected behavior.
For details, please check https://github.com/avocado-framework/avocado-vt/pull/3420.

Reply:
@Yingshun Thank you :smile:  ,Yingshun ,  but  Did you give the correct pr link

Reply:
@Yingshun Great .Thanks yingshun,I have updated it ,And will learn from this link.